pyproject.toml

```diff
@@ -28,6 +28,9 @@ platforms = ["any"]
 zip-safe  = true
 include-package-data = true
 
+[tool.setuptools.dynamic]
+dependencies  = {file = ["requirements.txt"]}
```

Comment:
It's not really good practice to mix development requirements and production requirements in the same file or list for libraries that should be used by other applications. For development we want to pin the requirements exactly to have reproducibility, but for production we want to be as forgiving as possible with the pinning of the library requirements to let applications avoid conflicts.

Reply:
OK, changed to not use dynamic dependencies